LearnositySDK/LearnositySDK.csproj

```diff
@@ -5,15 +5,15 @@
     <AssemblyName>LearnositySDK</AssemblyName>
     <RootNamespace>LearnositySDK</RootNamespace>
     <PackageId>LearnositySDK</PackageId>
-    <PackageVersion>0.9.1</PackageVersion>
+    <PackageVersion>0.10.0</PackageVersion>
     <Authors>Learnosity</Authors>
     <Description>Learnosity SDK for ASP.NET / C#</Description>
     <PackageReleaseNotes>Telemetry data (basic information about the execution environment) is now added to the request objects being signed which is later read and logged internally by our APIs when the request is received. This allows us to better support our various SDKs and does not send any additional network requests. More information can be found in README.md.
 </PackageReleaseNotes>
     <Copyright>Copyright Learnosity 2017 (c)</Copyright>
```

Comment:
Let's update the old date here from 2017 to this year, 2023 since this is when we are releasing it.